src/app/components/onboarding/login/login.component.ts

```diff
@@ -60,7 +60,12 @@ export class LoginComponent implements OnInit {
             if (error.status === 0) {
               this.errorMsg = this.translate.instant('CONNECTIONERROR');
             } else {
-              this.errorMsg = error.error.errorMessage.Message;
+              if (error.error.title) {
+                this.errorMsg = error.error.title;
+              }
+              if (error.error.errors.Email) {
+                this.errorMsg = error.error.errors.Email[0];
```

Comment:
Tipp: Hier würde es Sinn machen, das error array nach den formcontrol Namen zu durchsuchen und dann die Fehlermeldung am jeweiligen Control anzuzeigen

Reply:
Prinzipiell kann für jede Property des gesendeten Dtos eine Fehlermeldung zurückkommen

Reply:
Umsetzung in https://orso.atlassian.net/browse/ARPA-66